### PR TITLE
Adding keywords for kserve UI testing

### PR DESCRIPTION
**Pull request by mwaykole**

### CODE REVIEW

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource [hunk 1]
```diff
@@ -228,3 +228,76 @@ Open Dashboard Settings
...
+    END
+
+Click CheckBox Multi Mode Serving Platforms
+    [Documentation]    Enables Multi Mode serving platforms
+    ${status}=  Get Checkbox State Of Multi Mode serving platforms
+    IF  "${status}"=="False"
+        Select Checkbox   ${MULTI_MODE_SERVING_CHECK_BOX}
+    END
+
+UnClick CheckBox Single Mode Serving Platforms
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource:257]: it's Single Mode**l**

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource [hunk 2]
```diff
@@ -228,3 +228,76 @@ Open Dashboard Settings
...
+
+UnClick Both Model Serving Platforms
+    [Documentation]    Disbale Multi Mode and Single Mode serving platforms
+    UnClick CheckBox Single Mode Serving Platforms
+    UnClick CheckBox Multi Mode Serving Platforms
+
+Check IF Single Mode Model Serving Is Disabled From DSC And Cluster Setting
+    [Documentation]    Check single mode model should be disabled from DSC and dashboard
+    Component Should Not Be Enabled  kserve
+    ${status}   Get Checkbox State Of Single Mode Serving Platforms
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource:284]: could you also check that the checkbox is disabled/greyed out?

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource [hunk 3]
```diff
@@ -228,3 +228,76 @@ Open Dashboard Settings
     END
     Wait For RHODS Dashboard To Load    expected_page=${exp_page_title}
     ...    wait_for_cards=${FALSE}
+
+
+Get Checkbox State Of Multi Mode Serving Platforms
+    [Documentation]    Checks if  Multi model serving platform is enabled/checked in Cluster setting
+    ${is_checked} =  Run Keyword And Return Status  Checkbox Should Be Selected  ${MULTI_MODE_SERVING_CHECK_BOX}
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource:235]: `${MULTI_MODE_SERVING_CHECK_BOX}` where is it defined?

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource [hunk 4]
```diff
@@ -228,3 +228,76 @@ Open Dashboard Settings
...
+    [Return]  ${is_checked}
+
+Get Checkbox State Of Single Mode Serving Platforms
+    [Documentation]    Checks if  Single model serving platform is enabled/checked in Cluster setting
+    ${is_checked} =  Run Keyword And Return Status  Checkbox Should Be Selected  ${SINGLE_MODE_SERVING_CHECK_BOX}
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource:240]: `${SINGLE_MODE_SERVING_CHECK_BOX}` where is it defined?

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource (hunk 3 above)
Comment (mwaykole) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource:235]: thanks ill push it

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource (hunk 1 above)
Comment (mwaykole) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource:259]: sure ill change it

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource (hunk 2 above)
Comment (mwaykole) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource:286]: sure ill try to do it

**Review by lugi0 — CHANGES_REQUESTED** [review]
Other than Berto's comments, I would suggest changing `Mode` to `Model` in all keywords, otherwise LGTM

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource [hunk 5]
```diff
@@ -228,3 +230,82 @@ Open Dashboard Settings
...
+    [Documentation]    Check single model should be disabled from DSC and dashboard
+    Component Should Not Be Enabled  kserve
+    ${status}   Get Checkbox State Of Single Model Serving Platforms
+    Should Be Equal As Strings   ${status}  False
+    Element Should Be Disabled    ${SINGLE_MODEL_SERVING_CHECK_BOX}
+
+
+Check IF Single Model Serving Is Enabled From DSC And Cluster Setting
+    [Documentation]    Check single model should be enabled from DSC and dashboard
+    Component Should Not Be Enabled  kserve
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource:292]: `Component Should Be Enabled`

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource [hunk 6]
```diff
@@ -228,3 +230,82 @@ Open Dashboard Settings
...
+    [Documentation]    Check Multi model should be disabled from DSC and dashboard
+    Component Should Not Be Enabled  modelmeshserving
+    ${status}   Get Checkbox State Of Multi Model Serving Platforms
+    Should Be Equal As Strings    ${status}  False
+    Element Should Be Disabled    ${MULTI_MODEL_SERVING_CHECK_BOX}
+
+
+Check IF Multi Model Serving IS Enabled From DSC And Cluster Setting
+    [Documentation]    Check Multi model should be enabled from DSC and dashboard
+    Component Should Not Be Enabled  modelmeshserving
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource:308]: `Component Should Be Enabled`

**Review by bdattoma — CHANGES_REQUESTED** [review]

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource (hunk 6 above)
Comment (mwaykole) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource:308]: ahh thanks i missed to check it

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource [hunk 7]
```diff
@@ -228,3 +230,82 @@ Open Dashboard Settings
     END
     Wait For RHODS Dashboard To Load    expected_page=${exp_page_title}
     ...    wait_for_cards=${FALSE}
+
+Get Checkbox State Of Multi Model Serving Platforms
+    [Documentation]    Checks if  Multi model serving platform is enabled/checked in Cluster setting
+    ${is_checked} =  Run Keyword And Return Status  Checkbox Should Be Selected  ${MULTI_MODEL_SERVING_CHECK_BOX}
+    [Return]  ${is_checked}
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource:237]: please use `Return` instead of `[Return]`

**Review by bdattoma — APPROVED** [review]

**Review by bdattoma — APPROVED** [review]

**Review by lugi0 — APPROVED** [review]

**Review by tarukumar — APPROVED** [review]

**Pull request merged**